src/test/groovy/com/netflix/gradle/plugins/rpm/validation/RpmPackageNameAttributeValidatorTest.groovy

```diff
@@ -0,0 +1,32 @@
+package com.netflix.gradle.plugins.rpm.validation
+
+import spock.lang.Issue
+import spock.lang.Specification
+import spock.lang.Unroll
+
+class RpmPackageNameAttributeValidatorTest extends Specification {
+    RpmPackageNameAttributeValidator validator = new RpmPackageNameAttributeValidator()
+
+    @Issue("https://github.com/nebula-plugins/gradle-ospackage-plugin/issues/90")
+    @Unroll
+    def "verifies #description: '#attribute'"() {
+        when:
+        boolean valid = validator.validate(attribute)
+
+        then:
+        valid == result
+
+        where:
+        attribute       | result | description
+        'aBc_hello-1.0' | true   | 'valid package name with mixed alphanumeric characters'
```

Comment:
Could you add some more true examples? Possibly confusing on whether the 1.0 is a version or part of the package name.